apps/src/lib/node/ledger/shell/finalize_block.rs

```diff
@@ -100,17 +100,14 @@ where
                 namada_proof_of_stake::read_pos_params(&self.wl_storage)?;
             namada_proof_of_stake::copy_validator_sets_and_positions(
                 &mut self.wl_storage,
+                &pos_params,
                 current_epoch,
                 current_epoch + pos_params.pipeline_len,
             )?;
             namada_proof_of_stake::store_total_consensus_stake(
                 &mut self.wl_storage,
                 current_epoch,
             )?;
-            namada_proof_of_stake::purge_validator_sets_for_old_epoch(
-                &mut self.wl_storage,
-                current_epoch,
-            )?;
```

Comment:
where is the removal taking place now?

Reply:
the purges are now done internally in PoS `fn copy_validator_sets_and_positions` (https://github.com/anoma/namada/pull/1944/files#diff-4b7e249d17fe6a121cba4c86bf063ff3f8a57706ae9c56d746ad2ef0058039f2R1556) via calls to `Epoched/EpochedDelta::update_data`

Reply:
gotcha, thx!